logistics_planning_base/models/logistics_schedule.py

```diff
@@ -0,0 +1,83 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields
+
+TRANSPORT_TYPE = [
+    ('ground', 'Ground'),
+    ('ocean-going', 'Ocean-Going'),
+]
+
+class LogisticsSchedule(models.Model):
+    _name = 'logistics.schedule'
+    _description = 'logistics.schedule'
+
```

Comment:
We'll add a default order, .e.g.:
```suggestion
    _order = "scheduled_load_date desc, id desc"
```